spec/unit/lib/cloud_controller/diego/bbs_instances_client_spec.rb

```diff
@@ -6,26 +6,19 @@ module VCAP::CloudController::Diego
     let(:bbs_client) { instance_double(::Diego::Client) }
 
     describe '#lrp_instances' do
-      let(:bbs_response) { ::Diego::Bbs::Models::ActualLRPGroupsResponse.new(actual_lrp_groups: actual_lrp_groups) }
-      let(:actual_lrp_groups) { [actual_lrp_group] }
-      let(:actual_lrp_group) { ::Diego::Bbs::Models::ActualLRPGroup.new(instance: actual_lrp) }
+      let(:bbs_response) { ::Diego::Bbs::Models::ActualLRPsResponse.new(actual_lrps: actual_lrps) }
+      let(:actual_lrps) { [actual_lrp] }
       let(:actual_lrp) { ::Diego::Bbs::Models::ActualLRP.new(state: 'potato') }
       let(:process) { VCAP::CloudController::ProcessModelFactory.make }
       let(:process_guid) { ProcessGuid.from_process(process) }
 
       before do
-        allow(bbs_client).to receive(:actual_lrp_groups_by_process_guid).with(process_guid).and_return(bbs_response)
+        allow(bbs_client).to receive(:actual_lrps_by_process_guid).with(process_guid).and_return(bbs_response)
       end
 
       it 'sends the lrp instances request to diego' do
         client.lrp_instances(process)
-        expect(bbs_client).to have_received(:actual_lrp_groups_by_process_guid).with(process_guid)
-      end
-
-      it 'returns a resolved list of actual LRPs' do
```

Comment:
why was this test removed?

Reply:
There is no resolving required any more, you get the list of LRPs not groups, so no need to convert them from groups. `lib/diego/actual_lrp_group_resolver.rb` is removed.

Reply:
cool, thanks!